config/manifests/bases/assisted-service-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+    description: Assisted Service is used to orchestrate baremetal OpenShift installations.
+    operators.operatorframework.io/builder: operator-sdk-v1.3.0
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
+    "operatorframework.io/suggested-namespace": assisted-installer
+  name: assisted-service-operator.v0.0.1
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: Host
```

Comment:
The names have changes and not everything here is relevant. 
Need to deploy:
Agents.adi.io.my.domain
InstallEnv.adi.io.my.domain
Cluster deployment should come with hive.
Cluster and Image are not relevant anymore.

Reply:
I will update this. Should NMStateConfig be included?

@filanov Currently the CSV will include and install all the CRDs in this directory: https://github.com/openshift/assisted-service/tree/master/internal/controller/config/crd/bases. Images and Cluster are still defined here as well as ClusterDeployment from hive. Will these 3 CRDs be eventually removed from this repo? Alternatively I can change the CSV to be selective and not include those.

Reply:
Good question. in production clusterdeployment CRD should be deployed by hive. but for testing we probably need an option to deploy it as well.
InstallEnv and Agent will remain in this repo, we have a ticket to move them outside of internal dir. 
Regarding Cluster and Image we will do the cleanup this or next week.